ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplExternalTables.java

```diff
@@ -62,13 +62,21 @@
 
   private ReplExternalTables(){}
 
-  public static String externalTableLocation(HiveConf hiveConf, String location) {
-    String currentPath = new Path(location).toUri().getPath();
+  public static String externalTableLocation(HiveConf hiveConf, String location) throws SemanticException {
     String baseDir = hiveConf.get(HiveConf.ConfVars.REPL_EXTERNAL_TABLE_BASE_DIR.varname);
```

Comment:
if we are expecting the user to give this path with scheme and authority, then the default value should also be with scheme and authority

Reply:
ok